actions/views/rhs.js

```diff
@@ -280,6 +280,34 @@ export function selectPost(post) {
     };
 }
 
+export function updatePreviewOnCreateComment(showPreview) {
```

Comment:
`updatePreview...` has the connotation that we're persisting the preview itself. Might we transition to the terminology, `setShowPreview...` instead?